malcontent-control/application.c

```diff
@@ -215,6 +216,92 @@ mct_application_activate (GApplication *application)
   gtk_window_present (window);
 }
 
+static void
+mct_application_shutdown (GApplication *application)
+{
+  MctApplication *self = MCT_APPLICATION (application);
+  EmtrEventRecorder *recorder;
+  g_autoptr(MctManager) mct_manager = NULL;
+  g_autoptr(GSList) users = NULL;  /* (element-type ActUser) */
+  g_auto(GVariantBuilder) builder = G_VARIANT_BUILDER_INIT (G_VARIANT_TYPE ("aa{sv}"));
+  guint n_entries = 0;
+
+  /* Endless-specific code to send metrics containing all the parental controls
+   * for all users, so we can see how different parental controls features are
+   * being used.
+   *
+   * Serialise the app filter for each user. It’s OK to use blocking calls here,
+   * as the UI is no longer shown.
+   *
+   * See https://phabricator.endlessm.com/T28741#810046 */
+#define MCT_PARENTAL_CONTROLS_EVENT "449ec188-cb7b-45d3-a0ed-291d943b9aa6"
```

Comment:
Do you have plans to add this to https://phabricator.endlessm.com/w/software/metrics_events/currently_recorded_events/

Reply:
I do now :+1:

Reply:
https://phabricator.endlessm.com/phriction/diff/545/?l=41&r=42